src/integ_test_resources/ios/sdk/integration/cdk/cdk_integration_tests_ios/location_stack.py

```diff
@@ -0,0 +1,62 @@
+from aws_cdk import aws_iam, aws_location, core
+from common.common_stack import CommonStack
+from common.platforms import Platform
+from common.region_aware_stack import RegionAwareStack
+
+class LocationStack(RegionAwareStack):
+    def __init__(self, scope: core.Construct, id: str, common_stack: CommonStack, **kwargs) -> None:
+        super().__init__(scope, id, **kwargs)
+
+        self._supported_in_region = self.is_service_supported_in_region()
+
+        ##### Map #####
+
+        # Reusing the get_bucket_name function to get a unique name for the map
+        mapName = self.get_bucket_name("map")
```

Comment:
nit: If you can do so safely, you might refactor this method to something like `get_unique_resource_name` to clarify the intent